include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -861,6 +861,33 @@ def TTIR_SliceOp: TTIR_DPSOp<"slice"> {
     let hasVerifier = 1;
 }
 
+def TTIR_SelectOp: TTIR_DPSOp<"select"> {
+    let summary = "Select op.";
+    let description = [{
+      Extracts a sub-tensor (slice) from the input tensor along a specified dimension in few steps defined by the
+      `begin`, `length`, and `stride` attributes.
+      The `begin` specifies the start index for the selected dimension of the tensor.
+      The `length` specifies the number of elements to extract from the input tensor along the selected dimension.
+      The `stride` specifies the step size for the start index.
```

Comment:
Does `stride=0` mean no stride? If so, can you add that as a comment?

Reply:
Based on ForgeFE implementation of the op, it means no stride as it defaults to dimSize.